### PR TITLE
Add detailed build instructions for thrift and jar update

### PR DESCRIPTION
**Pull request by pi-r-p**
Tested on a ubuntu 21.10 fresh install.

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -1,10 +1,62 @@
...
+
 ## Install Thrift
 You need Thrift in order to build the Warp 10 platform.
-Currently, Warp 10 use Thrift release 0.11.0.
+Currently, Warp 10 uses Thrift 0.11.0, to be compatible with some Hbase libs. 
+All major distributions now ships Thrift 0.13+, so you need to compile Thrift.
+
+Make sure Thrift is not already installed:
+```bash
+sudo apt remove thrift-compiler
```
Comment (hbs) [inline BUILDING.md:17]: This is not needed, simply compile thrift with a specific target via --prefix=...

File: BUILDING.md [hunk 2]
```diff
@@ -1,10 +1,62 @@
...
+```bash
+cd
+sudo apt install git
+git clone https://github.com/senx/warp10-platform.git
+cd warp10-platform
+./gradlew pack
+```
+The jar file should be suffixed with the latest tag.
+```bash
+cp warp10/build/libs/warp10-$(git describe).jar /opt/warp10/bin/
```
Comment (hbs) [inline BUILDING.md:50]: This assumes you are building on the same machine where you are deploying, this is a very specific case. Instruct to copy the jar from warp10/build/libs to the bin directory of the Warp 10 deployment.

File: BUILDING.md [hunk 3]
```diff
@@ -1,10 +1,62 @@
...
+```
+
+Then, patch /opt/warp10/bin/warp10-standalone.sh with the version number you just compiled.
+```bash
+sed -i "s/WARP10_REVISION=.*/WARP10_REVISION=$(git describe)/g" /opt/warp10/bin/warp10-standalone.sh
```
Comment (hbs) [inline BUILDING.md:55]: Ditto, simply instruct to set the WARP10_REVISION to the correct value.

File: BUILDING.md [hunk 4]
```diff
@@ -1,10 +1,62 @@
...
 ## Install Thrift
 You need Thrift in order to build the Warp 10 platform.
-Currently, Warp 10 use Thrift release 0.11.0.
+Currently, Warp 10 uses Thrift 0.11.0, to be compatible with some Hbase libs. 
+All major distributions now ships Thrift 0.13+, so you need to compile Thrift.
+
+Make sure Thrift is not already installed:
+```bash
+sudo apt remove thrift-compiler
+sudo apt remove libthrift-java
```
Comment (hbs) [inline BUILDING.md:18]: ditto, leave the system as it is

File: BUILDING.md [hunk 5]
```diff
@@ -1,10 +1,62 @@
...
+sudo apt install openjdk-8-jdk
+```
+
 ## Install Thrift
 You need Thrift in order to build the Warp 10 platform.
-Currently, Warp 10 use Thrift release 0.11.0.
+Currently, Warp 10 uses Thrift 0.11.0, to be compatible with some Hbase libs. 
+All major distributions now ships Thrift 0.13+, so you need to compile Thrift.
+
+Make sure Thrift is not already installed:
```
Comment (hbs) [inline BUILDING.md:15]: It does not make any difference since we will be using a custom home for the new version.

**Review by hbs — CHANGES_REQUESTED** [review]

File: BUILDING.md (hunk 2 above)
Comment (pi-r-p) [reply BUILDING.md:49]: warning added.

File: BUILDING.md (hunk 3 above)
Comment (pi-r-p) [reply BUILDING.md:56]: warning added.

**Comment by pi-r-p** [comment]
All fixed.

**Review by Giwi — APPROVED** [review]

**Pull request merged**